dev/RadioButtons/RadioButtons.cpp

```diff
@@ -256,6 +264,16 @@ void RadioButtons::OnRepeaterElementClearing(const winrt::ItemsRepeater&, const
     if (auto const element = args.Element())
     {
         element.SetValue(s_childHandlersProperty, nullptr);
+
+        // If the removed element was the selected one, update selection to -1
+        if (auto const elementAsToggle = element.try_as<winrt::ToggleButton>())
+        {
+            auto const isChecked = elementAsToggle.IsChecked();
+            if(isChecked && isChecked.GetBoolean())
+            {
```

Comment:
This is repeated a few times -- should it be a helper?

Reply:
probably